ClassAttendance/src/ConsoleUI/Models/Lecturer/LecturerViewModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using BLL.Models;
+
+namespace ConsoleUI.Models.Lecturer
```

Comment:
ConsoleUI.ViewModels

Reply:
BLL should return business objects, isn't it? In UI i call all services i and build ViewModel in UI

Reply:
You not understand me.
The entity that I marked here - ViewModel.
But namespace doesn't illustrate this.
ConsoleUI.ViewModels - is a new folder where all view models should be alloacted.

Reply:
Fixed